nilearn/maskers/tests/test_nifti_maps_masker.py

```diff
@@ -573,3 +574,24 @@ def test_3d_images():
     assert epis.shape == (1, 3)
     epis = masker.fit_transform([epi_img1, epi_img2])
     assert epis.shape == (2, 3)
+
+
+@pytest.mark.skipif(
+    have_mpl, reason="Test requires matplotlib to be not installed."
```

Comment:
I guess that the canonical warning would be "Test requires matplotlib not to be  installed."

Reply:
Will fix this one and merge